amethyst_network/src/lib.rs

```diff
@@ -28,22 +29,24 @@ pub use {
     network_socket::NetSocketSystem,
 };
 
-use std::net::{SocketAddr, UdpSocket};
+use std::net::SocketAddr;
 
 use bincode::{deserialize, serialize, ErrorKind};
 use serde::{de::DeserializeOwned, Serialize};
+use laminar::net::UdpSocket;
+use laminar::Packet;
 
 /// Sends an event to the target NetConnection using the provided network Socket.
 /// The socket has to be bound.
-pub fn send_event<T>(event: &NetEvent<T>, target: &SocketAddr, socket: &UdpSocket)
+pub fn send_event<T>(event: &NetEvent<T>, addr: &SocketAddr, socket: &mut UdpSocket)
 where
     T: Serialize,
 {
     let ser = serialize(event);
     match ser {
         Ok(s) => {
             let slice = s.as_slice();
-            match socket.send_to(slice, target) {
+            match socket.send(&Packet::unreliable(*addr, slice.to_vec())) {
```

Comment:
I thought we moved to using slices?

Reply:
Our public API  not. I thought too. We can make an issue in laminar for this one.

Reply:
Yes, please open an issue in laminar and add a `TODO` with the issue number in laminar. so that we can keep track of this.

Reply:
It's going to be difficult since `Packet` would need a lifetime of function `send()` I think.